### ISSUE TITLE
[BITV] 9.4.1.2/2.4 - "Online status" Modal Dialog - The dialog has not been marked as such with the appropriate role of "dialog". (1)

### ISSUE THREAD

**Issue by AndyScherzinger** [nextcloud/server#37096]
The implementation also lacks the aria-modal="true" attribute. With the value "true", browsers and screen readers will be informed that the content contained in the dialog window is foreground and that other content on the web page should be ignored. More information can be found at https://www.hellbusch.de/dialogfenster-für-screemreader-als-modalkennzeichnen and https://www.w3.org/TR/wai-aria-practices/examples/dialog-modal/dialog.html.

- [x] https://github.com/nextcloud/nextcloud-vue/pull/3113

<details>
<summary>Details</summary>

https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-bfbf07e5c3-v2-n4

</details>

**Comment by Pytal** [comment]
Verified to be present on `master`, `stable26`, and `stable25`
